apps/dashboard/src/app/team/[team_slug]/[project_slug]/insight/components/BlueprintsPageHeader.tsx

```diff
@@ -0,0 +1,20 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { PlusIcon } from "lucide-react";
+
+export function BlueprintsPageHeader() {
+  return (
+    <div className="flex grow flex-col">
+      <div className="border-border border-b py-10">
+        <div className="container flex items-center justify-between">
+          <h1 className="font-semibold text-3xl tracking-tight">Blueprints</h1>
```

Comment:
The page's title "Blueprints" is confusing - "Insight" would be better

Reply:
changed it